NEWS

```diff
@@ -11,6 +11,171 @@
+* Introduced rcache distribution to VFS (5081c9d2f)
+#### Tools
+* Added support pre-registered memory in ucx_perftest (8125b882c)
+* Added loopback transport support for UCT perf tests (446ac0a80)
+* Print memory invalidate cap by ucx_info -d (d0675935d)
+### Bugfixes
+#### Core
+* Fixed not deallocating memory from ucp_mem_unmap if no rcache (ff3fbc7d1)
+* Fixed compiler warnings with -Og option (7617e45ae)
+* Fixed versioning infra (3cdd39f18, f5ca79c5c)
+* Multiple code improvements: refactoring, debug prints and assertions, etc (4198f9f51, 8bf6c1cfa, 287b1bb54, c3f310848, 2920e0c47, 24c74d10c, 03e9f2b00, 81ad07e90, bc1f1a0ac, 8db882ab1, 2a1def421, 672ac6e8b, cd016dd12, eb9a6b5c0, b2243a7b4, c4293983d, 76f1e4823, 865eca7db, 194ac47ac, 77cecb1e3, bb9101076, 050fae15a, 337dc62de, 2cc1efee0, b4e29e70e, 5083f56dc, 0ca0583ad, 830a91d46, ca3923b95, 28b3c26a1, 597d47124, e11d02b85, ab70ce3de, 622d6e038, 3bb649171, 76f1e4823, 572fea340, cc8260676, ea9922307, af3fc5a2c, 0ce726cd7, 76d9ed03e, bd81b941a, 7b4d154f4, 334072fe9, 2257b109a, 06af2d1fe, 5c564e1d4, 07c2f7b0d, c4293983d, 4b4a6976d, 0cfb0ab1b, e2a86a15a, 469705e87, 6ab76bf0b, 2a2c45387, e9a7c87d0, 7bafc7c20, d923167b1, 2cab571b1, ca57867dc, ec6f7af5c, 0d9060d2b)
+* Multiple improvements in build, test and docs infrastructure
+#### UCP
+* Fixed rkey_pack (edf068e85)
+* Multiple fixes in keepalive protocol (248143383, 2051fa53e, 95427f080, a24b8120e)
+* Fixed initialization request send state in case of SW RMA/AMO (f5941c3fd)
+* Fixed error handling in rma and bw lanes selection logic (05d71b689)
+* Fixed CM wireup fallback (231e28859)
+* Fixed occasional crash in finalize (c6c3594b6)
+* Fixed AM proto flags (7dc3d5c34)
+* Fixed single zcopy proto initialization for AM (94ece060c)
+* Fixed proto v2 selection, take into account user header length (c0e6e5273)
+* Fixed selecting auxiliary transports when creating EP for sending EP_REMOVED (759ba10a5)
+* Fixed printing invalid configuration (fedeee030)
+* Fixed allocation of indirect remote ID for internal EP if connected EP supports PEER_FAILURE (0d046e5be)
+* Fixed adding one lane to registration map with user memh (a9c5259e9)
+* Fixed memh allocation when no rcache (07c373dcf)
+* Fixed protocol selection logic for UCP AM send (d949934cc)
+* Fixed error handling flow for EP discard requests from pending queue (e8ec2b0d5)
+* Fixed EP destroy flow (c9d7b8ea9)
+* Disabled put-pipeline when sender side is host memory (eeace7cff)
+* Fixed rsc_index for prereg_md_map (ee5d6bcca)
+* Fixed wireup error handling flow Create EP which send WIREUP_MSG/EP_REMOVED with AM lane only (7a5da31bc)
+* Fixed parsing AM_ONLY flag from the header of a worker address (1c85313e3)
+* Fixed memtype_cache region info after merge (647242a2b)
+* Fixed probe for multi-fragment eager (1a76fb8c2)
+* Fixed alignment for AM rdesc init (e784b1255)
+* Fixed perf estimation for proto v2 (37e99203c)
+* Fixed ABI break of ucp_ep_params_t (888495078)
```

Comment:
we have never released this, right ?

Reply:
right, https://github.com/openucx/ucx/pull/7823#issuecomment-1003696923